tests/test_preprocessing.py

```diff
@@ -0,0 +1,56 @@
+from wsw.preprocessing import *
+from definitions import ROOT_DIR
+
+# for generating random test files
+import string
+import random
+
+# for audio file i/o
+import librosa
+import os
+import shutil
+
+
+def test_write_audio():
+    filename = librosa.ex('nutcracker')
```

Comment:
Please provide docstrings on what the unit tests are actually testing to allow the user to understand what's going on in each test.

Reply:
Done!